ui/effects/use-last-visible-item.js

```diff
@@ -4,6 +4,12 @@ import type { Node } from 'react';
 
 type InjectedItem = { node: Node, index?: number, replace?: boolean };
 
+/**
+ * Returns the index indicating where in the claim-grid to inject the ad element
+ * @param injectedItem
+ * @param listRef - A reference to the claim-grid
```

Comment:
Misleading.  In List mode, it's not even `claim-grid` ... it's a section.  
Anything that houses a bunch of `<li>` can be listRef.